utilities/tags/addTagsUsingCSV.py

```diff
@@ -0,0 +1,44 @@
+import csv
+import requests
+
+SEMGREP_API_KEY = ""
+API_URL = "https://semgrep.dev/api/v1/deployments/{deployment-slug}/projects/{project_name}/tags"
+# Path to your CSV file
+CSV_FILE = "projects_tags.csv"
+
+# Headers for API request
+def get_headers():
```

Comment:
I think this can be a constant here like it is in the other files, yeah? No need for a function.